src/sidebar/components/annotation-user.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const propTypes = require('prop-types');
+const { createElement } = require('preact');
+
+const { isThirdPartyUser, username } = require('../util/account-id');
+const { withServices } = require('../util/service-context');
+
+/**
+ * Display information about an annotation's user. Link to the user's
+ * activity if it is a first-party user or `settings.usernameUrl` is present.
+ */
+function AnnotationUser({ annotation, features, serviceUrl, settings }) {
+  const user = annotation.user;
+  const isFirstPartyUser = !isThirdPartyUser(user.userid, settings.authDomain);
+  const username_ = username(user);
+
+  // How should the user's name be displayed?
+  const displayName = (() => {
+    if (!isFirstPartyUser || !features.flagEnabled('client_display_names')) {
+      return username_;
+    }
+    if (annotation.user_info && annotation.user_info.display_name) {
+      return annotation.user_info.display_name;
+    }
+    return username_;
+  })();
+
+  const shouldLinkToActivity = isFirstPartyUser || settings.usernameUrl;
+
+  if (shouldLinkToActivity) {
+    return (
+      <div className="annotation-user">
```

Comment:
Since you are applying exactly the same styling to the `annotation-user` class and the link, you could just render the link directly in this case (without a containing `<div>`), styled with the `annotation-user` class. Then it becomes more obvious that the username is supposed to look the same in both cases.